.dockerignore

```diff
@@ -1 +1,4 @@
 node_modules
+
+# Package DB is created during docker build
+static/package-db.sqlite
```

Comment:
Remove this file from the commit

Reply:
I included it intentionally to make development easier, it's only about 20MiB.

Reply:
I also considered generating the files on `pnpm install` but that needs a certain set of dependencies in the system.